mycore-neo4j/src/main/java/org/mycore/mcr/neo4j/frontend/cli/MCRNeo4JCommands.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * This file is part of ***  M y C o R e  ***
+ * See http://www.mycore.de/ for details.
+ *
+ * MyCoRe is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MyCoRe is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with MyCoRe.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.mycore.mcr.neo4j.frontend.cli;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.mycore.common.config.MCRConfiguration2;
+import org.mycore.datamodel.common.MCRXMLMetadataManager;
+import org.mycore.datamodel.metadata.MCRMetadataManager;
+import org.mycore.datamodel.metadata.MCRObject;
+import org.mycore.datamodel.metadata.MCRObjectID;
+import org.mycore.frontend.cli.MCRAbstractCommands;
+import org.mycore.frontend.cli.annotation.MCRCommand;
+import org.mycore.frontend.cli.annotation.MCRCommandGroup;
+import org.mycore.mcr.neo4j.datamodel.metadata.neo4jparser.MCRNeo4JParser;
+import org.mycore.mcr.neo4j.utils.MCRNeo4JDatabaseDriver;
+import org.mycore.mcr.neo4j.utils.MCRNeo4JUtilsInterface;
+
+import java.util.List;
+
+import static org.mycore.mcr.neo4j.utils.MCRNeo4JDatabaseDriver.commitWriteOnlyQuery;
+
+@MCRCommandGroup(name = "Commands to handle Neo4J")
+@SuppressWarnings("unused")
+public class MCRNeo4JCommands extends MCRAbstractCommands {
+
+   private static final String UTIL_CLASS_PROPERTY = "MCR.Neo4J.Utils.Class";
+   private static final Logger LOGGER = LogManager.getLogger(MCRNeo4JCommands.class.getName());
+
+   @MCRCommand(syntax = "clean neo4j of id {0}", help = "clean the Neo4J database for an ID {0}", order = 10)
```

Comment:
Ich hätte hier `clean neo4j for id` als Kommando besser gefunden.
In anderen Kommandos haben wir aber beide Varianten verwendet.

Reply:
... ist mir auch egal, nehmen wir for :)

Reply:
soll ich hier generell alle of durch for tauschen? oder nur bei den beiden Methoden `clean neo4j of id {0}` und `synchronize neo4j of id {0}` ?

Reply:
Ich würde alles ersetzen `for id`, `for type` ,...